src/tvheadend/entity/Entity.h

```diff
@@ -34,7 +34,7 @@ namespace tvheadend
     class Entity
     {
     public:
-      Entity() : m_dirty(false), m_id(0) {};
+      Entity() : m_id(0), m_dirty(false) {};
```

Comment:
If you find yourself wondering what to do on a Friday night you could perhaps move this change to a separate commit ;-)